src/components/Modals/FiatRamps/views/Manager.tsx

```diff
@@ -88,6 +90,20 @@ const ManagerRouter: React.FC<ManagerRouterProps> = ({ fiatRampProvider }) => {
   const {
     state: { wallet },
   } = useWallet()
+  const accountIds = useAppSelector(state =>
+    selectPortfolioAccountIdsByAssetId(state, { assetId: selectedAsset?.assetId ?? '' }),
+  )
+  const [accountId, setAccountId] = useState<AccountId | null>(null)
+
+  useEffect(() => {
+    if (!accountIds.length) return
+    setAccountId(accountIds[0])
+  }, [accountIds])
```

Comment:
we can't do this - it's dangerous as it may not match what the user has selected

Reply:
This is basically for the initial value, and since it's the same as the AccountsDropdown logic ai think we're good to go. But if you think it's not correct, I'll change it.

Reply:
Nice catch, my initial take on that was that it's fine since it reacts on a reference of `accountIds` that will never change, and that was confirmed by testing, but after giving this second thought, we should remove this `useEffect()` call altogether, because 
1. the order of hooks call could make this invalid
2. the default this is supposedly setting is already happening somewhere else:

`handleAccountIdChange` is guaranteed to fire at least once, even if a user didn't select an account, with the default `AccountId` from the dropdown, effectively making this hook duplicate logic

Reply:
we **only** ever want `accountId` to be set via the callback from the dropdown, any other logic is a second source of truth and a source of bugs. we have to be super careful with this stuff as it deals with send/receive addresses

Reply:
@stackedq we can use that logic in the dropdown because we need to visually render something, and it is the source of truth. duplicating that logic is dangerous.